<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="whisperTab">
      <attribute name="title">
       <string>Whisper 模型设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>本地 Whisper 设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="whisperPathLabel">
            <property name="text">
             <string>Whisper 程序路径：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="whisperPathLineEdit">
            <property name="placeholderText">
             <string>自动检测或手动指定路径</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="browseWhisperPathButton">
            <property name="text">
             <string>浏览...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="modelSizeLabel">
            <property name="text">
             <string>模型大小：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="modelSizeComboBox">
            <item>
             <property name="text">
              <string>tiny</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>base</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>small</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>large</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="languageLabel">
            <property name="text">
             <string>识别语言：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="languageComboBox">
            <item>
             <property name="text">
              <string>auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>en</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>zh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ja</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ko</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>fr</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>de</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>es</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ru</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="downloadModelButton">
            <property name="text">
             <string>下载 Whisper 模型</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>在线 API 设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="preferOnlineApiCheckBox">
            <property name="text">
             <string>优先使用在线 API 进行识别</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="apiUrlLabel">
              <property name="text">
               <string>API 地址：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="apiUrlLineEdit">
              <property name="text">
               <string>https://api.example.com/asr</string>
              </property>
              <property name="placeholderText">
               <string>在线语音识别 API 地址</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="subtitleTab">
      <attribute name="title">
       <string>字幕设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>字幕保存设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="subtitleDirLabel">
            <property name="text">
             <string>保存目录：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="subtitleDirLineEdit">
            <property name="placeholderText">
             <string>字幕文件保存路径</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="browseSubtitleDirButton">
            <property name="text">
             <string>浏览...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>重置为默认值</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>